### PR TITLE
Add test for URL rendering

### PR DESCRIPTION
**Pull request by kfdm**
Similar to our previous PR, we want to add some starting test cases to our url rendering paths so that we are prepared for future updates.

### CODE REVIEW

File: promgen/templates/promgen/navbar.html [hunk 1]
```diff
@@ -36,7 +36,7 @@
                         <li><a href="{% url 'api:api-root' %}">API</a></li>
                         <li><a href="{% url 'api:all-targets' %}">Export Targets</a></li>
                         <li><a href="{% url 'api:all-rules' %}">Export Rules</a></li>
-                        <li><a href="{% url 'config-urls' %}">Export URL</a></li>
+                        <li><a href="{% url 'api:all-urls' %}">Export URL</a></li>
```
Comment (vincent-olivert-riera) [inline promgen/templates/promgen/navbar.html:39]: I know it's unrelated to your PR, but while at it, would you mind fixing the grammatical number of "URL"?

"Export URL" makes the user think that only one URL will be exported. If you don't mind, could you please change it to "Export URL**s**" (in plural)?

**Review by vincent-olivert-riera — APPROVED** [review]
Looks good to me. I just made an optional request.

File: promgen/templates/promgen/navbar.html (hunk 1 above)
Comment (kfdm) [reply promgen/templates/promgen/navbar.html]: Sure, updated and force pushed

**Review by vincent-olivert-riera — APPROVED** [review]

File: promgen/templates/promgen/navbar.html (hunk 1 above)
Comment (vincent-olivert-riera) [reply promgen/templates/promgen/navbar.html]: Re-approved. Thank you very much 👍

**Pull request merged**